src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BucketHelper.scala

```diff
@@ -29,7 +32,9 @@ class BucketHelper(dataprocConfig: DataprocConfig,
                    googleComputeDAO: GoogleComputeDAO,
                    googleStorageDAO: GoogleStorageDAO,
                    serviceAccountProvider: ServiceAccountProvider)
-                  (implicit val executionContext: ExecutionContext) extends LazyLogging {
+                  (implicit val executionContext: ExecutionContext) extends LazyLogging with Retry {
+
+  val system = ActorSystem("BucketHelperActor")
```

Comment:
Defining ^^ feels weird. I could add it as an `implicit` to `BucketHelper` constructor and see if it'd be provided by `Boot` or such...or is there a better way?

Reply:
Yeah I don't think we should define a new `ActorSystem` -- would be better to pass one into `BucketHelper`'s constructor.